src/xrpld/ledger/OpenView.h

```diff
@@ -156,7 +168,11 @@ class OpenView final : public ReadView, public TxsRawView
         : OpenView(open_ledger, &*base, rules, base)
     {
     }
-    /** @} */
+
+    OpenView(batch_view_t, OpenView& base) : OpenView(std::addressof(base))
+    {
+        baseTxCount_ = base.txCount();
```

Comment:
Remind me why this is needed? Couldn't figure it out from a quick poke around. Is it for the tx queue? I thought we weren't counting the inner transactions there.

Reply:
txCount() is used in metadata as well. (ApplyStateTable)